docs-standards/20-style-the-content/06-bullet-lists.md

```diff
@@ -0,0 +1,28 @@
+# Bullet lists
+
+- Don't use a bullet list for actions that users should complete in an orderly manner; use numbered lists in procedures.
+- Use a bullet list to improve the readability of sentences or paragraphs where you list or describe three or more items.
+
+    | Usage      | Example                                                                                           |
+    |:------------|:---------------------------------------------------------------------------------------------------|
+    | **Correct**| You need a machine running Ubuntu Linux with the following minimum requirements:<br>- 4 GB memory<br>- 2 TB SSD<br>- Linux 64-bit |
+    | Incorrect  | You need a machine running Ubuntu Linux with the following minimum requirements: 4 GB memory, 2 TB SSD, and Linux 64-bit. |
+    | Incorrect  | You need a machine with 4 GB of memory, a 2 TB SSD, and a 64-bit Ubuntu Linux operating system.  |
+
+- Use a colon (:) before the bullet list items.
+- Use periods at the end of each sentence in a bullet list for [punctuation](./05-punctuation-and-symbols.md). Using periods on a bullet list can be tricky. Use the following guidelines:
+    - When the list items are complete sentences, use a period at the end of each sentence.
+    - Don't use a period when the list items are not complete sentences or sentences with three words or less.
+    - In a list with elements of more and less than three words mixed, don't use a period.
+    - If one item in a column is a complete sentence, all items in that column should end with a period, even if they are a single word.
```

Comment:
Added this rule based on this discussion: https://github.com/status-im/ift-docs/discussions/39